protected/humhub/modules/space/views/spaces/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2021 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+use humhub\assets\DirectoryAsset;
+use humhub\modules\space\components\SpaceDirectoryQuery;
+use humhub\modules\space\widgets\SpaceDirectoryCard;
+use humhub\modules\space\widgets\SpaceDirectoryFilters;
+use humhub\widgets\Button;
+use humhub\widgets\LinkPager;
+use yii\web\View;
+
+/* @var $this View */
+/* @var $spaces SpaceDirectoryQuery */
+
+DirectoryAsset::register($this);
+?>
+<div class="panel panel-default">
+
+    <div class="panel-heading">
+        <?= Yii::t('SpaceModule.base', '<strong>Space</strong> directory'); ?>
+    </div>
+
+    <div class="panel-body">
+        <?= SpaceDirectoryFilters::widget(); ?>
+    </div>
+
+</div>
+
+<div class="row cards">
+    <?php if (!$spaces->exists()): ?>
+    <div class="col-md-12">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <?= Yii::t('SpaceModule.base', 'No spaces found!'); ?>
+            </div>
+        </div>
+    </div>
+    <?php endif; ?>
+
+    <?php foreach ($spaces->all() as $space) : ?>
+        <?= SpaceDirectoryCard::widget(['space' => $space]); ?>
+    <?php endforeach; ?>
+</div>
+
+<?php if (!$spaces->isLastPage()) : ?>
+<div class="directory-load-more">
+    <?= Button::primary(Yii::t('SpaceModule.base', 'Load more'))
+        ->icon('fa-angle-down')
+        ->sm()
```

Comment:
No `sm` ?

Reply:
Do you want to remove `->sm()` from here and make the button to standard size?

Reply:
Yes

Reply:
https://github.com/humhub/humhub/pull/5057/commits/25c012131efec873609b19774315b0b6011f9ffe